GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg

```diff
@@ -0,0 +1,99 @@
+@PART[ROE-HG3]:NEEDS[RealPlume]:BEFORE[RealPlume]
+{
+	@MODULE[ModuleEngineConfigs]
+	{
+		%type = ModuleEnginesRF
```

Comment:
This seems a strange thing to do in a plume patch?

Reply:
What seems strange is that I just copied the plume from ROE J-2, so it's there too